buildkite/scripts/run-test.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -euo pipefail
+
+eval $(opam env)
+export PATH=/home/opam/.cargo/bin:/usr/lib/go/bin:$PATH
+export GO=/usr/lib/go/bin/go
+
+# make build_or_download_pv_keys 2>&1
+
+# Waiting for the coda-daemon image to be built by CircleCI (this is redundent once build-artifacts works in buildkite)
+
+git_tag=$(git describe --abbrev=0)
+git_branch=$(echo $BUILDKITE_BRANCH | sed 's!/!-!; s!_!-!g')
+# git_branch=$(git rev-parse --symbolic-full-name --abbrev-ref  HEAD  | sed 's!/!-!; s!_!-!g')
+git_hash=$(git rev-parse --short=7 HEAD)
+
+coda_daemon_image="codaprotocol/coda-daemon:${git_tag}-${git_branch}-${git_hash}"
+export TF_LOG=DEBUG
+
+echo "image: ${coda_daemon_image}"
+
+# echo "Waiting for coda-daemon image to appear in docker registry"
+# for i in $(seq 60); do
+#     docker pull "$coda_daemon_image" && break
+#     [ "$i" != 30 ] || (echo "coda-daemon image never appeared in docker registry" && exit 1)
+#     sleep 60
+# done
+
+echo "--- Building test-executive"
+
+dune build --verbose --profile=testnet_postake_medium_curves src/app/test_executive/test_executive.exe
+
+echo "--- Clone coda-automation repo (this would be changed in future. Nathan is suggesting linking coda-automation in coda repo)"
```

Comment:
Just a note: we can remove all of the redundancy below by running the `dune build...` step above once, uploading it as an artifact and then allowing all tests cases to pull it and execute on agents (without each test having to build its own).